aspnetcore/signalr/get-started-signalr-core/sample/Startup.cs

```diff
@@ -0,0 +1,53 @@
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using SignalRCoreChat.Hubs;
+
+namespace SignalRCoreChat
+{
+    public class Startup
+    {
+        public Startup(IConfiguration configuration)
+        {
+            Configuration = configuration;
+        }
+
+        public IConfiguration Configuration { get; }
+
+        // This method gets called by the runtime. Use this method to add services to the container.
+        public void ConfigureServices(IServiceCollection services)
+        {
+            services.AddMvc();
+            services.AddSignalR();
+        }
+
+        // This method gets called by the runtime. Use this method to configure the HTTP request pipeline.
+        public void Configure(IApplicationBuilder app, IHostingEnvironment env)
+        {
+            if (env.IsDevelopment())
+            {
+                app.UseBrowserLink();
+                app.UseDeveloperExceptionPage();
+            }
+            else
+            {
+                app.UseExceptionHandler("/Error");
+            }
+
+            app.UseStaticFiles();
+
+            app.UseSignalR(routes =>
+            {
+                routes.MapHub<ChatHub>("chat");
```

Comment:
More than once, I have accidentally created a `Chat.cshtml` razor page and a `chat` Hub (and I **really** should know better 😝). I've also seen customers do it. Of course, when that happens, one of them doesn't work (which one depends on the order of SignalR and MVC in `Configure`) because they are both mapped to the same route.

Can we start a precedent of mapping hubs to `/hubs/[HubName]` in our docs to ensure we keep Hubs in their own area of the URL space? (Thoughts @davidfowl ?).